juriscraper/opinions/united_states/state/ga.py

```diff
@@ -15,69 +14,59 @@ class Site(OpinionSiteLinear):
-    def _process_html(self):
-        paths = "//p/strong | //p/b | //p/font/strong | //p/font/b"
-        for date_element in self.html.xpath(paths):
-            string = date_element.xpath("./text()")
-            try:
-                string = string[0]
-                # handle examples where time but no date (ga_example_3.html)
-                if ":" in string and ("AM" in string or "PM" in string):
-                    continue
-                # handle legacy example (ga_example.html)
-                string = string.split("SUMMARIES")[0]
-                date_string = re.sub(r"\W+", " ", string)
-                # handle legacy example (ga_example.html)
-                if len(date_string.split()) != 3:
-                    continue
-            except:
-                continue
-            parent = date_element.xpath("./..")[0]
-            # handle legacy example (ga_example.html)
-            while parent.tag != "p":
-                parent = parent.xpath("./..")[0]
-            for item in parent.getnext().xpath("./li"):
-                text = item.text_content()
-                if text:
-                    # Extract Docket numbers
-                    dockets = re.findall(self.regex_docket, text)
-                    if not dockets:
-                        if (
-                            text
-                            == "IN RE: MOTION TO AMEND 2021-3 (Bar Rule 1.2)"
-                        ):
-                            continue
-                        else:
-                            raise InsanityException(
-                                f"Could not find docket numbers in: '{text}'"
-                            )
+        :param year: Year to scrape
+        :return: URL for the given year
+        """
+        return f"https://www.gasupreme.us/opinions/{year}-opinions/"
 
-                    # Extract name substring; I am sure this could
-                    # be done with a more slick regex, but its not
-                    # my forte...
-                    name = text
-                    for docket in dockets:
-                        name = name.replace(docket, "")
-                    name = name.lstrip(" .,")
+    def _process_html(self) -> None:
+        for link in self.html.xpath("//li/a[contains(@href, '.pdf')]"):
+            url = link.get("href")
+            title = link.text_content()
+            if not title:
+                continue
+            dockets = re.findall(r"S?\d{2}\w\d{4}", title)
+            if not dockets:
+                # Skip empty links in the HTML
+                continue
+            docket = ", ".join(dockets)
+            name = title.split(dockets[-1])[-1].strip("., ")
+            summary = link.xpath(
+                ".//parent::li/parent::ul/preceding-sibling::p[1]"
+            )[0].text_content()
+            # We've got en/em dash and hyphens.
+            date_str = (
+                summary.split("–")[0].split("—")[0].split("-")[0].strip()
+            )
```

Comment:
An explanation would help a lot here.

Reply:
Added examples and comments about the characters used as separator between the dates and the summary text.